weblate/settings_docker.py

```diff
@@ -1502,3 +1502,6 @@
     with open(ADDITIONAL_CONFIG) as handle:
         code = compile(handle.read(), ADDITIONAL_CONFIG, "exec")
         exec(code)  # noqa: S102
+
+# Asset domains
+ALLOWED_ASSET_DOMAINS = ["*"]
```

Comment:
Please add the default value to some AppConf instance so that it doesn't have to be present in the settings. `weblate/utils/models.py` might be a good place.